boards/samd21-xpro/include/periph_conf.h

```diff
@@ -70,6 +71,12 @@ extern "C" {
 #define CLOCK_PLL_DIV       (1U)                /* adjust to your needs */
 /* generate the actual used core clock frequency */
 #define CLOCK_CORECLOCK     (((CLOCK_PLL_MUL + 1) * 1000000U) / CLOCK_PLL_DIV)
+#elif CLOCK_XOSC32_DFLL
```

Comment:
Can this option be CLOCK_USE_XOSC32_DFLL so it marries with the other option better? I get lost in all of the clock related #defines here and the USE token helps me group them together.